crates/bevy_ui/src/ui_node.rs

```diff
@@ -1844,22 +1844,31 @@ impl TargetCamera {
     }
 }
 
+#[derive(Component)]
+/// Marker used to demarcate default cameras, they will have priority over `PrimaryWindow` cameras
```

Comment:
It would be good to have a bit more context on why this might be useful in these docs.

Reply:
Added the new context!